lib/cfonb/operation_detail/mmo.rb

```diff
@@ -15,7 +15,7 @@ def self.apply(operation, line)
 
         operation.original_amount = sign * BigDecimal(line.detail[4..17]) / (10**scale)
         exchange_rate_value = line.detail[26..29]
-        return unless exchange_rate_value
+        return if exchange_rate_value.nil? || exchange_rate_value.empty?
```

Comment:
what about this ?
`
exchange_rate_value = line.detail[26..29].to_i
return if exchange_rate_value.zero?
`

Reply:
to_i and to_f is not working here.
I added specs. pls review.
do you know what `04` means here `EUR200000001875625             04` ?

Reply:
I downloaded all statements and parsed them. This is what we have an operation line. I can send them to you.